.github/workflows/auto_assign_action.yml

```diff
@@ -0,0 +1,13 @@
+name: Auto Assign Issues
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Auto-assign issue'
+        uses: pozil/auto-assign-issue@v1.1.0
+        with:
+          assignees: jdemelo-splunk,pzhou-splunk,rgil-splunk
```

Comment:
I took a look at https://github.com/marketplace/actions/auto-assign-issue and `assignees` is described as

> Comma separated list of **user** names. Issue will be assigned to those users.

My username is `jdemelo` even though the display name is `jdemelo-splunk`. It's possible this action could work with either, but just wanted to mention that.

Reply:
oh huh, I never even noticed that 😅 . I'll update it just in case